specification/redis/Cache.Management/PrivateEndpointConnection.tsp

```diff
@@ -0,0 +1,65 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Cache;
+/**
+ * The Private Endpoint Connection resource.
+ */
+@parentResource(RedisResource)
+model PrivateEndpointConnection
+  is Azure.ResourceManager.ProxyResource<PrivateEndpointConnectionProperties> {
+  ...ResourceNameParameter<
+    Resource = PrivateEndpointConnection,
+    KeyName = "privateEndpointConnectionName",
+    SegmentName = "privateEndpointConnections",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface PrivateEndpointConnections {
+  /**
+   * Gets the specified private endpoint connection associated with the redis cache.
+   */
+  get is ArmResourceRead<
+    PrivateEndpointConnection,
+    Response = ArmResponse<Azure.ResourceManager.CommonTypes.PrivateEndpointConnection>
+  >;
+
+  /**
+   * Update the state of specified private endpoint connection associated with the redis cache.
+   */
+  put is ArmResourceCreateOrReplaceAsync<
+    PrivateEndpointConnection,
+    LroHeaders = ArmLroLocationHeader & Azure.Core.Foundations.RetryAfterHeader
```

Comment:
>ArmLroLocationHeader [](http://example.com/codeflow?start=17&length=20)

Should this be `ArmLroLocationHeader<FinalResult = PrivateEndpointConnection>`? Does it make any difference?